contents/blog/how-to-treat-your-pricing-like-a-product.md

```diff
@@ -0,0 +1,102 @@
+---
+date: 2023-04-09
+title: How to treat your pricing like a product (and why it matters)
+rootPage: /blog
+sidebar: Blog
+showTitle: true
+hideAnchor: true
+author:
+  - raquel-m-smith
+featuredImage: ../images/blog/happy-hog.png
+featuredImageType: full
+category: Product growth
+tags:
+  - Product engineers
+  - Guides
+
+---
+"Pricing is important" is common knowledge – price determines profit margin, which directly or indirectly determines nearly everything else. 
+
+What isn't common knowledge is twofold:
+
+1. Your pricing should be as much of a product as the thing you actually market and sell
+2. How to go about treating your pricing like a product
+
+Let’s start with number 1.
+
+## Your pricing should be a product
+
+I feel like this statement is easy to nod along with yet difficult to grok. What does it actually mean to treat your pricing like a product? What even is a product? Isn’t everything a product?
+
+Yes, kind of. But we’re talking specifically about software products, and I bet you know them when you see them – even if you’re not paying attention to the specifics. 
+
+Here’s a rough working version of how I might define a “product”:
+
+- Products are akin to living, breathing things that are constantly being changed and improved. New features are added, unused features are (hopefully) removed, and the last commit wasn’t 4 years ago.
+
+- Products are tested – definitely from a code standpoint, and hopefully from a product standpoint. 
+
+- Products have customers who require them to do a certain job. Those customers can be external or internal, and there is some sort of exchange of value for using the product.
+
+It’s easy to see what the products are for nearly any company – at PostHog our analytics suite, session replay, feature flag, and A/B testing tools are all products. They fit the definition above perfectly. 
+
+But pricing is often doesn’t fit this definition. It’s often left static and unchanged for years at a time, it’s not tested from a code or product standpoint, and people don’t even realize that they are customers of this thing because it feels so dead.
```

Comment:
```suggestion
But pricing often doesn’t fit this definition. It’s often left unchanged for years at a time, it’s not tested from a code or product standpoint, and people don’t realize that they are customers of it because it feels so dead.
```